linera-views/src/test_utils/mod.rs

```diff
@@ -452,7 +452,10 @@ pub async fn run_test_batch_from_blank<C: LocalKeyValueStore>(
     batch: Batch,
 ) {
     let kv_state = realize_batch(&batch);
-    key_value_store.write_batch(batch, &[]).await.unwrap();
+    key_value_store
+        .write_batch(batch.clone(), &[])
```

Comment:
I fail to see the point of this `clone`.

Reply:
Left in from debugging, I think — thanks!